ivy/functional/backends/tensorflow/experimental/activations.py

```diff
@@ -66,3 +66,12 @@ def silu(
     if ivy.exists(out):
         return ivy.inplace_update(out, ret).astype(x.dtype)
     return ivy.astype(ret, x.dtype)
+
+
+@with_supported_dtypes({"2.12.0 and below": ("float",)}, backend_version)
+def elu(x: Tensor, /, *, alpha: float = 1.0, out: Optional[Tensor] = None) -> Tensor:
+    alpha = tf.cast(alpha, x.dtype)
+    ret = tf.cast(tf.where(x > 0, x, alpha * tf.expm1(x)), x.dtype)
```

Comment:
This line is breaking the tests as we call `tf.math.expm1` not `tf.expm1`

Reply:
hey @hirwa-nshuti  i have changed the tf.expm1 to tf.math.expm1 and also implemented the elu function for paddle backend. in the paddle backend i called the elu function from paddle.nn.function class so i implemented the elu function in paddle.nn.functional as well with its test function. you can check and if there any mistake please let me know.

thank you